samples/Common.c

```diff
@@ -564,6 +566,8 @@ STATUS createSampleStreamingSession(PSampleConfiguration pSampleConfiguration, P
     audioTrack.kind = MEDIA_STREAM_TRACK_KIND_AUDIO;
     audioTrack.codec = RTC_CODEC_OPUS;
     audioRtpTransceiverInit.direction = RTC_RTP_TRANSCEIVER_DIRECTION_SENDRECV;
+    audioRtpTransceiverInit.rollingBufferDurationSec = 1;
+    audioRtpTransceiverInit.rollingBufferBitratebps = 5 * 1024 * 1024;
```

Comment:
For audio transceiver from my research even for professional studio quality recording we will not exceed 1.5Mbps.  I think we should use that as a default in the sample.

Reply:
I have switched the default to be based on the codec. For opus, based on the official website: https://opus-codec.org/#:~:text=It%20can%20scale%20from%20low,)%20to%2048%20kHz%20(fullband) --> it seems the range is 6-51kbps

For H.264, given we use 720p in our GStreamer sample and based on multiple resources I looked at, the range seems to be 1.5-4 Mbps. So set the value to 4Mbps.

I updated the graphs in the description to show memory consumption with these settings too.

I also increased default buffer duration to 3 seconds to be safe.